data/mods/letsgo/moves.ts

```diff
@@ -83,8 +83,6 @@ export const BattleMovedex: {[k: string]: ModdedMoveData} = {
 	},
 	teleport: {
 		inherit: true,
-		desc: "If this move is successful and the user has not fainted, the user switches out even if it is trapped and is replaced immediately by a selected party member. The user does not switch out if there are no unfainted party members.",
-		shortDesc: "User switches out.",
```

Comment:
You seem to be removing these and then not doing anything with them later.

Reply:
Yeah, these should be added back in.